### PR TITLE
fix: fix migrate to jina 3

### PR DESCRIPTION
**Pull request by guenthermi**
This PR should refactor the executer to be executable with Jina 3.

### CODE REVIEW

File: tests/integration/test_with_flow.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def test_integration(request_size: int):
             return_results=True,
         )
 
-    assert sum(len(resp_batch.docs) for resp_batch in resp) == 50
+    assert len(resp) == 50
```
Comment (makram93) [inline tests/integration/test_with_flow.py:30]: I can not select it here, but in the line number 23., we don't need the `return_results=True` param. It should by default return the `DocumentArray`

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ matches = [
 question.matches.extend(matches)
 
 f = Flow().add(
-    uses='jinahub+docker://DPRReaderRanker', uses_with={'num_spans_per_match': 1}
+    uses='jinahub+docker://deaReaderRanker', uses_with={'num_spans_per_match': 1}
```
Comment (makram93) [inline README.md]: I think this is mis-spelt?

Comment (JoanFM) [inline README.md]:
```suggestion
    uses='jinahub+docker://DPRReaderRanker', uses_with={'num_spans_per_match': 1}
```

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**